pyproject.toml

```diff
@@ -44,6 +44,7 @@ flake8 = "^4.0.1"
 
 # These are solely for work on the example notebooks, and not required for
 # the package itself.
+cvrplib = "^0.1.1"

```

Comment:
This is fine by me, but can we pull the instance from local (that is, from inside the repo)? I don't want to force users to have access to the internet to work with examples, nor have a dependency on a website that may or may not exist five years down the road.